CMake/ctr/n3ds_defs.cmake

```diff
@@ -15,7 +15,6 @@ set(TTF_FONT_DIR \"romfs:/\")
 
 #Force scaling, for now..

```

Comment:
I'm guessing this comment is no longer true?

Reply:
That's correct. Thanks for catching that.